dev/ci/presubmits/test-e2e-repo-agent

```diff
@@ -22,22 +22,34 @@ from shared import utils
 
 
 def main():
-    """ invokes e2e tests in kind cluster and outputs a junit report in the ARTIFACTS dir
-
-    The ARTIFACTS environment variable is set by prow.
+    """ invokes e2e tests in kind cluster
     """
     repo_root = utils.get_repo_root()
+    working_dir = repo_root + "/repo-agent"
+    print(f"using working dir: {working_dir}")
 
     result = subprocess.run([f"{repo_root}/dev/tools/create-kind-cluster", "e2e-test", "--recreate", "--kubeconfig", f"{repo_root}/bin/KUBECONFIG"])
```

Comment:
The script executes a series of commands and stops at the first failure. It would be beneficial to add more logging before each step to make it easier to debug which command failed and why.